### PR TITLE
Refresh config fields

### PR DESCRIPTION
**Pull request by jimmyxian**
Refresh config fields
Signed-off-by: Xian Chaobo <xianchaobo@huawei.com>

### CODE REVIEW

**Comment by jimmyxian** [comment]
ping @aluzzardi @vieux @donhcd

**Comment by vieux** [comment]
LGTM

File: types.go [hunk 1]
```diff
@@ -11,28 +11,31 @@ type ContainerConfig struct {
...
 	Volumes         map[string]struct{}
+	VolumeDriver    string
 	WorkingDir      string
 	Entrypoint      []string
 	NetworkDisabled bool
+	MacAddress      string
 	OnBuild         []string
+	Labels          map[string]string
+
+	// FIXME: Compatibility
```
Comment (donhcd) [inline types.go]: what's this referring to?

**Comment by donhcd** [comment]
mostly looks good to me, just confused about the comment ^

**Comment by jimmyxian** [comment]
....My bad,

**Comment by donhcd** [comment]
got it. LGTM

**Pull request merged**